### PR TITLE
Switch to Rails

### PR DESCRIPTION
**Pull request by gabebw**
I visited the homepage and every link in the header (About, Donate, Resources).
They are all exactly the same as the design on railsbridgeboston.org (I
switched back and forth between tabs to double check).

### CODE REVIEW

**Comment by gabebw** [comment]
Uhhhh what if I don't fix any of these problems in autogenerated files?

File: spec/lib/eventbrite_spec.rb [hunk 1]
```diff
@@ -25,19 +25,20 @@
...
+          status: 500
+        )
       end
     end
 
     def stub_successful_eventbrite_request
-      response = { events: [{ "title" => "event info" }] }
+      events = { events: [{ "title" => "event info" }] }
 
-      stub_request(:get, eventbrite_api_url).to_return(body: { id: "123" })
```
Comment (gabebw) [inline spec/lib/eventbrite_spec.rb]: `stub_request` with the same method and URL overwrites the stub each time, so this stub was never working

File: spec/lib/eventbrite_spec.rb [hunk 2]
```diff
@@ -25,19 +25,20 @@
       end
 
       def stub_failed_eventbrite_request
-        stub_request(:get, eventbrite_api_url).to_return(body: { id: "123" })
```
Comment (gabebw) [inline spec/lib/eventbrite_spec.rb]: This stub was overwritten by the next one, so I folded it into the next one

File: Gemfile [hunk 3]
```diff
@@ -1,24 +1,61 @@
...
+gem "httparty"
+gem "i18n-tasks"
 gem "jquery-middleman"
+gem "jquery-rails"
+gem "neat", "~> 1.7.0"
+gem "newrelic_rpm", ">= 3.9.8"
 gem "nokogiri"
-gem "httparty"
+gem "normalize-rails", "~> 3.0.0"
+gem "pg"
```
Comment (mxie) [inline Gemfile]: We're not actually using a database, can we omit this?

File: Gemfile [hunk 4]
```diff
@@ -1,24 +1,61 @@
...
+gem "rack-contrib"
+gem "rails", "4.2.1"
+gem "recipient_interceptor"
```
Comment (mxie) [inline Gemfile]: I don't think we have to worry about emails either.

File: README.md [hunk 5]
```diff
@@ -1,33 +1,32 @@
-RailsBridge Boston Website
-==========================
+# Railsbridgeboston dot org
```
Comment (mxie) [inline README.md]: Let's revert this header back to the original.

File: README.md [hunk 6]
```diff
@@ -1,33 +1,32 @@
...
 
-[![Build
```
Comment (mxie) [inline README.md]: Let's keep this in!

File: README.md [hunk 7]
```diff
@@ -1,33 +1,32 @@
...
-Status](https://travis-ci.org/railsbridge-boston/railsbridgeboston_dot_org.svg?branch=master)](https://travis-ci.org/railsbridge-boston/railsbridgeboston_dot_org)
+## Getting Started
 
-This is the website for RailsBridge Boston, a chapter of [RailsBridge]. You can
```
Comment (mxie) [inline README.md]: We should keep these informational paragraphs.

File: Gemfile [hunk 8]
```diff
@@ -1,24 +1,54 @@
...
+source "https://rubygems.org"
 
-ruby '2.1.0'
+ruby "2.2.2"
 
-gem "middleman", "~> 3.2.2"
-gem "middleman-livereload"
-gem "middleman-dotenv"
-gem "rack-contrib"
-gem "compass", "~> 1.0.0.alpha.20"
```
Comment (gabebw) [inline Gemfile]: We're not actually using Compass.

File: app/assets/stylesheets/_resources.scss [hunk 9]
```diff
@@ -1,4 +1,4 @@
-.resources_index {
+.resources {
```
Comment (mxie) [inline app/assets/stylesheets/_resources.scss]: Wait, was this intentional?

File: app/views/pages/about.html.erb [hunk 10]
```diff
@@ -37,10 +37,10 @@ title: About - RailsBridge Boston
   <section class="contact">
     <h2 class="section-title">Contact</h2>
     <ul>
-    <% data.links.contacts.each do |contact| %>
-        <li class="contact-<%= contact.type %>">
-          <%= contact.description %>
-          <a href="<%= contact.url %>"><%= contact.title %></a>
+    <% data_links["contacts"].each do |contact| %>
```
Comment (mxie) [inline app/views/pages/about.html.erb]: Indent 1 more level in.

File: app/views/pages/about.html.erb [hunk 11]
```diff
@@ -49,10 +49,10 @@ title: About - RailsBridge Boston
   <section class="links-out">
     <h2 class="section-title">Our Friends</h2>
     <ul>
-      <% data.links.friends.each do |friend| %>
+      <% data_links["friends"].each do |friend| %>
         <li>
-          <a href="<%= friend.url %>"><%= friend.name %></a>:
-          <%= friend.description %>
+          <a href="<%= friend["url"] %>"><%= friend["name"] %></a>:
```
Comment (mxie) [inline app/views/pages/about.html.erb]: How about `link_to` here instead?

File: app/views/pages/about.html.erb [hunk 12]
```diff
@@ -37,10 +37,10 @@ title: About - RailsBridge Boston
...
     <h2 class="section-title">Contact</h2>
     <ul>
-    <% data.links.contacts.each do |contact| %>
-        <li class="contact-<%= contact.type %>">
-          <%= contact.description %>
-          <a href="<%= contact.url %>"><%= contact.title %></a>
+    <% data_links["contacts"].each do |contact| %>
+        <li class="contact-<%= contact["type"] %>">
+          <%= contact["description"] %>
+          <a href="<%= contact["url"] %>"><%= contact["title"] %></a>
```
Comment (mxie) [inline app/views/pages/about.html.erb]: `link_to`?

File: README.md (hunk 6 above)
Comment (gabebw) [inline README.md]: oooooops

File: app/assets/stylesheets/_resources.scss (hunk 9 above)
Comment (gabebw) [inline app/assets/stylesheets/_resources.scss]: Yes, it fixes the design and has no side effects that I could see.

Middleman gives one of the pages a class of `resources resources_index`, so I gave the appropriate page in Rails a class of `resources`. I could have used `resources_index` too, I suppose.

File: spec/lib/eventbrite_spec.rb (hunk 2 above)
Comment (mxie) [inline spec/lib/eventbrite_spec.rb]: Oops. That totally makes sense. What was I thinking?

File: app/assets/stylesheets/_resources.scss (hunk 9 above)
Comment (mxie) [inline app/assets/stylesheets/_resources.scss]: Either way is fine, as long as we're consistent and not breaking all the pretty styles Connie has put into place!

Comment (gabebw) [inline app/assets/stylesheets/_resources.scss]: They are pixel-perfectly exact.

**Comment by mxie** [comment]
This looks pretty good Gabe! The only other thing is that I'd like to remove anything that involves the database. I can't foresee a situation where we might need it, so I don't think it makes sense to keep that stuff around. Thoughts?

**Comment by gabebw** [comment]
I think I removed everything around the DB - I searched for `activerecord` and `active_record` and removed those settings, and also disabled the AR Railtie. I removed DelayedJob and PG from the Gemfile and removed all of their configuration too.

**Comment by gabebw** [comment]
Was that a provisional :+1:, Melissa?

**Comment by gabebw** [comment]
@mxie

**Comment by gabebw** [comment]
OK, I'm merging.

**Pull request merged**